ably/rest/rest.py

```diff
@@ -41,22 +44,38 @@ def __init__(self, options=None):
           - `auth_url`: Undocumented
           - `keep_alive`: use persistent connections. Defaults to True
         """
+        hasKey = key or (options and options.keyValue and options.keyId)
+        hasMeansToFetchToken = options and ( options.authUrl  or options.auth_callback)
+        hasToken = token or (options and options.auth_token)
+        if token and options and options.auth_token and token != options.auth_token:
```

Comment:
I am not sure why we support token and authToken. See https://github.com/ably/ably-java/blob/master/src/io/ably/types/ClientOptions.java and https://github.com/ably/ably-java/blob/master/src/io/ably/rest/Auth.java#L44-L97, and documentation http://docs.ably.io/client-lib-development-guide/features/#options